### PR TITLE
FEAT: Toast toggling

### PR DESCRIPTION
**Pull request by sw0ok**
Necessary for later stages, and to fix toast popping up on Connection/Archive screens.

### CODE REVIEW

File: src/hook/context.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import React from "react";
-import ToastContainer from "../toast-container";
+import React from 'react';
+import ToastContainer from '../toast-container';
 
 export type ToastType = Pick<
-  ToastContainer,
-  "show" | "update" | "hide" | "hideAll" | "isOpen"
+    ToastContainer,
+    'show' | 'update' | 'hide' | 'hideAll' | 'isOpen' | 'toggle'
```
Comment (MrBlackmax) [inline src/hook/context.ts]: toggle sounds like its calling either show or hide, maybe call it toggleToastsEnabled or something more descriptive

**Review by MrBlackmax — CHANGES_REQUESTED** [review]
rename the function pls

File: src/hook/context.ts (hunk 1 above)
Comment (sw0ok) [reply src/hook/context.ts]: Yeah, changed to toggleToastVisibility and state to isVisible

**Review by MrBlackmax — APPROVED** [review]
OK

**Pull request merged**